Tasks/PackageDownloader/task.json

```diff
@@ -0,0 +1,110 @@
+{
+  "id": "8d6e8f7e-267d-442d-8c92-1f586864c62f",
+  "name": "PackageDownloader",
+  "friendlyName": "Package Downloader",
+  "description": "Download package from package management.",
+  "helpMarkDown": "",
```

Comment:
+1
It might also be helpful informing the users that only NuGet is supported at the moment.